lib/api/controllers/documentController.js

```diff
@@ -72,11 +73,14 @@ function DocumentController (kuzzle) {
     assertBodyAttributeType(request, 'ids', 'array');
     assertHasIndexAndCollection(request);
 
+    if (request.input.body.ids.length > kuzzle.config.server.maxMultiActionsCount) {
```

Comment:
@scottinet I agree, if someone need to retrieve a list of documents, the best way is to use search API with `ids` filters